lightwood/mixers/nn.py

```diff
@@ -155,7 +155,8 @@ def _fit(self, train_ds, test_ds, stop_training_after_seconds=None):
             input_size=len(input_sample),
             output_size=len(output_sample),
             nr_outputs=len(train_ds.out_types),
-            deterministic=self.deterministic
+            deterministic=self.deterministic,
+            dropout=CONFIG.DEFNET_DROPOUT if not self.deterministic else None
```

Comment:
Any reason for this to be in the `CONFIG` now that you're done experimenting ?

If so, it should be in the mixer config (part of the larger config that is passed when creating a lightwood predictor)

Reply:
Nope, no reason, good catch.